### DISCUSSION TITLE
support for 3dconnexion space mouse...

### DISCUSSION THREAD

**Discussion by harrisjono** [FreeCAD/FreeCAD#12790] [Discussions that will turn in to issues but need refining]
system set-up: 
Ubuntu 23.10
freecad 0.21.2
blender 4.0

i can download and run blender 4.0(on either of my linux boxes) and it will recognize my spaceMouse compact, but freecad will not.

i am not into C++, but, here are the  locations for blender's spacenav cmake files for reference
1. blender/build_files/cmake/Modules/FindSpacenav.cmake
https://github.com/blender/blender/blob/main/build_files/cmake/Modules/FindSpacenav.cmake

2. blender/build_files/build_environment/cmake/spnav.cmake
https://github.com/blender/blender/blob/main/build_files/build_environment/cmake/spnav.cmake

is there any way to give freecad native-support /automatic-recognition of spaceMouse products? 

thank you.

**Comment by luzpaz** [comment]
Lots of discussions on the forum regarding this. Can you find them and bump them ?

**Comment by maurerle** [comment]
It works fine for me with a 3dconnexion compact in combination with freecad and blender on debian trixie:

```
blender/testing,testing,now 4.3.2+dfsg-2 amd64 [installed]
freecad/testing,testing,testing,testing,now 1.0.0+dfsg-6 all [installed]
spacenavd/testing,testing,now 1.2-3 amd64 [installed]
```

FreeCad About:

```
OS: Debian GNU/Linux trixie/sid (GNOME/gnome/wayland)
Architecture: x86_64
Version: 1.0.0.
Build type: Release
Python 3.13.2, Qt 5.15.15, Coin 4.0.3, Vtk 9.3.0, OCC 7.8.1
Locale: English/United States (en_US)
Stylesheet/Theme/QtStyle: unset/FreeCAD Classic/Qt default
```
